src/App.js

```diff
@@ -1,8 +1,180 @@
+import {React, useState} from 'react'
+import './App.css'
+import styled from 'styled-components';
+import TodoInput from "./components/TodoInput.js";
+import TodoList from "./components/TodoList.js";
+import DoneList from "./components/DoneList.js";
+let getId = 1;
+
+const Container = styled.div`
+    background-color: black;
+    width: 30rem;
+    margin: auto;
+    height: 50rem;
+    border-style: solid;
+    border-radius: 2%;
+    border-color: white;
+`
+
+const Title = styled.div`
+    width: 90%;
+    margin-left: 5%;
+    margin-top: 3rem;
+    margin-bottom: 1.5rem;
+    height: 5vh;
+    color: white;
+    font-size: 30px;
+    text-align: center;
+    font-weight: 800;
+
+    border-bottom: solid;
+    border-radius: 2%;
+`
+
+const TodoTitle = styled. div`
+    width: 82%;
+    height: 32px;
+    color: white;
+    font-size: 28px;
+    text-align: left;
+    font-weight: bold;
+    margin-left : 15%;
+    margin-top : 1.5rem;
+    margin-bottom: 4px;
+    display: flex;
+`
+const DoneTitle = styled. div`
+    width: 82%;
+    height: 32px;
+    color: white;
+    font-size: 28px;
+    text-align: left;
+    font-weight: bold;
+    margin-left : 15%;
+    margin-top : 1.5rem;
+    margin-bottom: 4px;
+    display: flex;
+`
+const TodoBox = styled.div`
+    position: relative;
+    width: 82%;
+    height: 30%;
+    margin-left: 8%;
+    margin-top: 1rem;
+`
+
+const DoneBox = styled.div`
+    position: relative;
+    width: 82%;
+    height: 30%;
+    margin-left: 8%;
+    margin-top: 1rem;
+`
+const TodoRldBtn = styled.button`
+    margin-left: 50%;
+    border: none;
+    width: 60px;
+    font-size : 25px;
+    background-color : black;
+`
+const DoneRldBtn = styled.button`
+    margin-left: 50%;
+    border: none;
+    width: 60px;
+    font-size : 25px;
+    background-color : black;
+`
+
 function App() {
+  const [todos, setTodos] = useState([]);
+  const addTodoList = (text) => {
+    if (text === ""){ //trim 나중에 적용시키기
+      return alert("입력된 내용이 없습니다.");
+    } else{
+      const todo = {
+        id: getId,
+        text,
+        checked: false
+      }
+      setTodos(todos => todos.concat(todo));
```

Comment:
예지님 말씀처럼 spread(...) 연산자를 사용하는 게 좋을 것 같아요!

```suggestion
      setTodos([...todos,todo]);
```